bci_tester/data.py

```diff
@@ -544,17 +551,14 @@ def create_BCI(
 
 REPOCLOSURE_CONTAINER = DerivedContainer(
     base="registry.fedoraproject.org/fedora:latest",
-    containerfile=r"""RUN dnf -y install 'dnf-command(repoclosure)'
+    containerfile=f"""RUN dnf -y install 'dnf-command(repoclosure)'
```

Comment:
you can combine `r` and `f`, so no need for the `\` escaping then
```suggestion
    containerfile=rf"""RUN dnf -y install 'dnf-command(repoclosure)'
```

Reply:
yeah, don't particularly like raw strings, but I'm not gonna argue about it..